ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/StatefulSessionComponent.java

```diff
@@ -235,6 +235,10 @@ public AccessTimeoutDetails getAccessTimeout(Method method) {
     }
 
     public SessionID createSession() {
+        // check to see that StatefulSessionComponent.stop() has not been called concurrent with the request; this can happen on undeploy (see WFLY-5178)
+        if (getCache() == null)
```

Comment:
I guess seeing that the method is defined in the class where this.cache is an instance variable, I should just access it directly ...